kubernetes/build.gradle

```diff
@@ -38,9 +40,13 @@ task createRDockerfile(type: Dockerfile) {
 
 task createPythonDockerfile(type: Dockerfile) {
   destFile = outputFilePython
-  from "spark-py:${sparkVersion}"
+  arg "spark_version"
+  from 'spark-py:$spark_version'
+  arg "spark_version"
   copyFile("h2o_pysparkling_${sparkMajorVersion}-${version}.zip", "/opt/spark/pyspark/python/lib/h2o_pysparkling_${sparkMajorVersion}-${version}.zip")
-  runCommand "pip install /opt/spark/pyspark/python/lib/h2o_pysparkling_${sparkMajorVersion}-${version}.zip"
+  runCommand 'echo "pyspark==$spark_version" > constraints'
+  runCommand "pip install /opt/spark/pyspark/python/lib/h2o_pysparkling_${sparkMajorVersion}-${version}.zip -c constraints"
```

Comment:
this line instlalls all depencies including pyspark. Shouldn't pyspark be omitted if it's already in the image?

Reply:
Yup, great catch! 

I would just verify after this change that the image works as expected -> starting pysparkling works